evap/evaluation/tools.py

```diff
@@ -130,7 +136,7 @@ def clean_email(email: EmailT) -> EmailT:
     return email
 
 
-def capitalize_first(string: str) -> str:
+def capitalize_first(string: StrOrPromise) -> str:
```

Comment:
Ah interesting; should we write a comment that clarifies that this realizes the lazy promise object?

Reply:
added a docstring